src/lib.rs

```diff
@@ -18,3 +18,4 @@ pub use uefi::UefiBoot;
 
 const KERNEL_FILE_NAME: &str = "kernel-x86_64";
 const RAMDISK_FILE_NAME: &str = "ramdisk";
+const CONFIG_FILE_NAME: &str = "config.json";
```

Comment:
How about naming it `boot.json`? This way, kernels could use their own `config.json` file.